packages/base-styles/_z-index.scss

```diff
@@ -105,7 +105,7 @@ $z-layers: (
 	".components-popover": 1000000,
 
 	// ...Except for popovers immediately beneath wp-admin menu on large breakpoints
-	".components-popover.block-editor-inserter__popover": 99998,
+	".components-popover.block-editor-inserter__popover": 99999,
```

Comment:
.components-popover is higher above? Should we just remove this rule?

Thinking here when I see all the popover exceptions. I do wonder why we need all these exceptions, why popovers shouldn't always appear above everything? (Aside, the "block-toolbar" popover where I understand that it should appear under the editor header.)

That said, I feel right now, any change to this file is completely random and while it fixes something might break something else.

Reply:
I very much agree there's an opportunity here for some simplification. Not sure if that's the best next step, though, or whether this added exception is a good interim fix, even if I agree.

Reply:
To clarify, I'm fine with shipping this because it doesn't put us in a worse situation, it's just as bad as today :P. A small concern about potential regressions we're missing though.

Reply:
@ellatrix any thoughts?